pkg/object/dummy.go

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2023 Alibaba Cloud, Inc. or its affiliates.
```

Comment:
The CLA means all the contributed code will be owned by Juicedata Inc, we won't accept any code copyrighted by others, except existing library.